### DISCUSSION TITLE
GPU problem

### DISCUSSION THREAD

**Discussion by tiomaldy** [coqui-ai/TTS#637] [General Q&A]
![image](https://user-images.githubusercontent.com/61519589/125176455-1a27e480-e199-11eb-91c8-1fd53dc6719e.png)
I need some help i don t know what are the command that i need to put to run this. i tried

python ./train_tts.py --config_path /Users/Gustavo/Desktop/Entrenar/config.json  CUDA_VISIBLE_DEVICES="0"

CUDA_VISIBLE_DEVICES="0"  python ./train_tts.py --config_path /Users/Gustavo/Desktop/Entrenar/config.json

both don t work

**Comment by erogol** [comment]
There is no path like ./train_tts.py but TTS/bin/train_tts.py

Also what does "don't work"mean? What error does it cause?

**Comment by erogol** [comment]
I see you started training in another thread 👍
